SPECS/selinux-policy/selinux-policy.spec

```diff
@@ -38,61 +54,61 @@ enforced by the kernel when running with SELinux enabled.
 %dir %{_sysconfdir}/selinux
```

Comment:
I tried to install both selinux-policy and libsemanage and noticed a conflict:
`file /etc/selinux from install of selinux-policy-2.20210203-1.cm1.noarch conflicts with file from package libsemanage-2.9-4.cm1.x86_64`
Changelog from libsemanage looks like it expects selinux-policy to `Requires: libsemanage` to resolve this.

Reply:
Fixed.